pyproject.toml

```diff
@@ -149,7 +149,7 @@ semantic_version = "^2.8.4"
 sphinx = "==1.8.2"
 sphinx-autobuild = "==0.7.1"
 sphinxcontrib-programoutput = "==0.11"
-pygments = "==2.2.0"
+pygments = "==2.6.1"
```

Comment:
@alwx Why is this pinned to a specific version?

Reply:
Probably there is no reason for that — all the docs-related things were just copied from the old requirements-docs